### ISSUE TITLE
Unable to use with `@emotion/css`

### ISSUE THREAD

**Issue by u3u** [ony3000/prettier-plugin-classnames#73]
## Dependency information

```
["ERROR" - 1:19:58 PM] Error formatting document.
["ERROR" - 1:19:58 PM] The second format failed. This is likely a bug in this plugin. Please file an issue.
Error: The second format failed. This is likely a bug in this plugin. Please file an issue.
    at Object.parse (/node_modules/.pnpm/prettier-plugin-classnames@0.7.1_prettier-plugin-astro@0.14.1_prettier@3.3.3/node_modules/prettier-plugin-classnames/dist/index.js:2:32837)
```

## Steps to reproduce

There will be problems when using `css` in `class`.

The `css` is a special name that we should ignore, Prettier has built-in support for recognizing `emotion` / `styled-components` for css format.

Example: [Prettier Playground](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBLAtgBwgJxgAmHwBM4AzVKOAYQi2gQIF98yc78ByAAQDcBXOAHpI9AM6oYqaAFoAhplScAOlAzY8hfDADu+Fmw6ddlYirX1NRMGLH7W7dF25x0EKdBG3zquAA8NAlIyOX4AGyDySho6bCpYAApgVXx8KDl0OCQuAFEsGABPTgAaVRT8HARSHASASkJy1MqYfhwofATG1PwAHkwu7vwwMLlbAF5gAG0Bwe0dAAMAIwg-GWWcavwdAAtJODFMOTA4GUxKmTDo-EwZAAZ8CEOwSQKZAA5b+dL22cHdeZgOCOAGtKABzGSTW4AOgAjJg-ABdfBhOByYjgyGw6EANmR8D8MBkx1gcBw2n8RMmAGIAOwAVgZDMRXxmgxsYnmbNmADIkEhFuRcHBvr8xeL8HykHIyPByckfhKld1sBIPFBsnJFmIIGF+PAANzc5WzGCPbK3I2Kk0S1Gyi1Wm3KjEHEYFbKUS5UNZhCBgYGOp0SnSoYgwbbZWG3W4AUkDQbF2zgqDB2xgkejceNNqY2ZNUsFZGFDWtCfZ0HgsGynC48bLg0WILB7H4UGIxN1uGy1LIvbr9dSjf9zYgrfblUwaPTaQgMgnU-7A6HwJHY9OEDV0g1Q0YZMX9eXq7bMgkAC8skMILw93mbY8ji8LdC3vuy4eW8eMHIwRfWmEEkIAAkwCHDAcpQGIky8IiTAAPy8GMwHhqgYjQvw-ChkwtSvk6ualmWPLQkcUjXvyMpyiWA6zEWsAnqg56RgATAiOFBl6JxJimabZIxbwsbeJqQL6ODZLwcg1DIMjhq4Ir4NSjG3HIty9thAnKgSRJyJcYLbiScqsU6NFEiE6CoGE7r4KgkBQEZBk2tZlbTioyguTktIAMw5JwdkmkZMg6MmqbTh8lpqeKeFKqy+GpDBMwAHzcsAyGoYBYi+jAqHBKEEQRbMPRCJgCXWrU5RMN8WGqCAxQgI86piMgoDiewOgAAriQg9UoFpOhyAU9XVYsQL+nAMAAMoZHAAAy0TICEYRiCKICDSCI2jU84LIICgjVQtpmbTg20gP4k44BgjBaQAKmS6Snfss1aQtO3gqiACK-DuHA93zYtABWYh+KNz1wG9H1fY9IAAI7vfALXsJgnUgKMMhUHApDEFVICAnIZngrQ6DoHIyCI2EYQYxIUBgqiACCYGnYs+pwC1ZLTVQYOLWm6BhAA6rs8AHEccCjR1kioLwLxExyGMCHAACSbaMKNYCnZgMBU22o2FKibPVWcG5wFzQKYETZz7GS14Y5QC14LD34E9rICHDgVtE4Uk5iErqAqxjZyUDAXOhuGyAfNVlRQ6glQ22CdtIHN4MwFq-thtsyCMdV-ALRdWqdbHi2uIKxBo5NcgU-w35wAAYrgBNgRtXX6hAIBMEwQA)

```jsx
import { defineComponent } from '@vue/composition-api'
import { tw } from 'twind'
import { css } from '@emotion/css'

export default defineComponent({
  name: 'Empty',

  render() {
    return (
      <p
        class={[
          tw`box-border whitespace-pre-line p-0 opacity-80`,
          tw`tracking-[0.1px] leading-[1.6] text-center text-[#757575]`,

          css`
            font-size: 15px;
            // ^^^^^^^^^^^^^ Cause error
          `,
        ]}
      >
        {this.$slots.default}
      </p>
    )
  },
})

```

## The current behavior

<img width="956" alt="image" src="https://github.com/user-attachments/assets/3a24e6b3-8142-49f1-a37c-3c8e0522227a">

Formatting error occurred, or when there is no error, the content in `css` will be compressed into one line, losing the formatting effect.

## The expected behavior

The `css` is a special name that we should ignore, Prettier has built-in support for recognizing `emotion` / `styled-components` for css format.

**Comment by ony3000** [comment]
Oh, I rarely use `styled-components` or similar libraries, so I missed this use case.

Currently, there is a workaround by using the `css` syntax outside of the `class` attribute and assigning the result to a variable.

```javascript
import { defineComponent } from '@vue/composition-api'
import { tw } from 'twind'
import { css } from '@emotion/css'

const style = css`
  font-size: 15px;
`;

export default defineComponent({
  name: 'Empty',

  render() {
    return (
      <p
        class={[
          tw`box-border whitespace-pre-line p-0 opacity-80`,
          tw`tracking-[0.1px] leading-[1.6] text-center text-[#757575]`,

          style,
        ]}
      >
        {this.$slots.default}
      </p>
    )
  },
})
```

**Comment by u3u** [comment]
Yes, putting it externally can solve the problem, but many old files were written like this before. If I happen to need to modify it, I can only temporarily skip the formatting.

**Comment by ony3000** [comment]
@u3u Fixed in `v0.7.3`, check it out!
